tests/test_atom_mapping.py

```diff
@@ -368,24 +367,18 @@ def test_connected_core_with_large_numbers_of_cores():
     )
     assert len(all_cores) > 0
 
-    # If this takes more then 30 minutes, something is off.
-    assert time.time() - start < 1800.0
-
-
-def get_mol_name(mol) -> str:
-    """Return the title for the given mol"""
-    return mol.GetProp("_Name")
+    # If this takes more then a minute, something is off.
+    assert time.time() - start < 60.0
```

Comment:
This test now runs in a couple of seconds rather than 30 minutes. Reduced the threshold to ensure we don't go back to a really slow implementation.